CHANGELOG.md

```diff
@@ -33,6 +33,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] - ReleaseDate
 
+### Changed 🛠
+- [PR#4](https://github.com/Rust-GPU/spirt/pull/4) updated `SPIRV-Headers`
+  to match Vulkan SDK 1.3.290
+
+### Fixed 🩹
+- [PR#2](https://github.com/Rust-GPU/spirt/pull/2) included a combination of control-flow (restructurization) refactors and fixes:
+  - fixed rewrites of chains of `OpPhi`s (for e.g. Rust-GPU's non-minimal SSA form)
+  - isolated the `ControlRegion`s of unstructured SPIR-T CFGs from eachother
+    - i.e. removed the implicit dominance-based (SSA-style) "value visibility" semantics
+    - explicit (phi-like) propagation of *all* values across CFG edges required
+    - structured control-flow, however, retained its more flexible "value visibility"
+    - (re)structurization "fuses" away the massively redundant dataflow *when sound*
+  - fixed SSA dominance issues (introduced by [PR#48 "minimal loops"](https://github.com/EmbarkStudios/spirt/pull/48)) using the above
+    - i.e. opportunistically simplify conservative (maximally redundant) dataflow
+    - could be expanded on in the future, moving towards hermetic (RVSDG-like) regions
+  - fixed issues with `ExitInvocation`s, by supporting them in structured SPIR-T
+    - e.g. this includes terminators like `OpKill`, `OpEmitMeshTasksEXT`, etc.
+    - also enables (very hacky) "abort" via `ExitInvocation(OpReturn)` from entry-point
+  - optimized common cases of divergent (`unreachable`/`ExitInvocation`) control-flow
+    - e.g. `if cond { abort() } else { foo() }` -> `if cond { abort() } else {} foo()`
+    - effectively flattens chains of checks (w/ the `abort() = ExitInvocation(OpReturn)` hack)
+
+---
+
+### ⬆️ *above entries after repository move (to [`Rust-GPU/spirt`](https://github.com/Rust-GPU/spirt)*)
+*See also [the transition announcement blog post](https://rust-gpu.github.io/blog/transition-announcement/).*
+### ⬇️ *below entries before repository move (from [`EmbarkStudios/spirt`](https://github.com/EmbarkStudios/spirt))*
+
+---
```

Comment:
@Rust-GPU/rust-gpu-maintainers This is what I am most worried about.
<sub>(most of the other additions are about all the changes that landed together in #2, which I'm glad I was able to summarize, but are not super critical - tho feel free to comment on them as well, ofc)</sub>

You can [preview the rendered CHANGELOG](https://github.com/LykenSol/spirt/blob/release-0.4.0/CHANGELOG.md) to see what it will look like.

Here it might be less important, but I hope we can come up with something like this for the `rust-gpu` repo as well.

I've also considered changing all the `PR#123` below this point to say "PR<sub>Embark</sub>" or otherwise indicate the distinction, but that felt overkill.

Reply:
I wouldn't overthink it, this looks fine to me.

Reply:
Suggestion: Include the repository transition notice at the very top so it's the first thing you see, and note that all PRs >40 are from embark's repo and everything <40 is from this repo. I think that's easy enough to understand and immediately visible, especially in the rust-gpu repo where embark got to >1000 PRs.
Also you don't have to write two changelogs or copy the splitter multiple times, if we had embark and rust-gpu PRs in the same category.

Reply:
Ideally we will have over 40 PRs eventually though...

Reply:
But not for this release, right? I would have just added the notice to this particular release, as any further releases would obviously be PRs from this repo, not embarks

Reply:
Yeah, sure. Again, I don't think this matters one way or another as the only consumer of spir-t is rust-gpu ha.